node/node-sdk.md

```diff
@@ -0,0 +1,610 @@
+# NodeJS SDK Proposal
+
+NodeJS is a perfect candidate for authoring Temporal workflows, by leveraging V8 isolates for running workflow code,
+we can create a JS runtime that's completely deterministic.
+
+The SDK steers developers to write their workflows and activities in TypeScript but vanilla JS is also supported.
+All examples in this proposal are written in TypeScript.
+
+## Minimal Example
+#### `activities/greeter.ts`
+```ts
+export async function greet(name: string) {
+  return `Hello, ${name}!`;
+}
+```
+
+#### `interfaces/workflows.ts`
+```ts
+import { Workflow } from '@temporal-sdk/interfaces';
+
+export interface Example extends Workflow {
+  greet(name: string): Promise<void>;
```

Comment:
On my first read-through this confused me for a sec - I thought for some reason we were re-declaring activity signatures. Perhaps calling this `main` a-la the later example would be more clear. (or maybe even something super explicit like `workflowEntrypoint`

Reply:
That's a mistake, it should be called `main`.